cumulus/polkadot-parachain/src/main.rs

```diff
@@ -14,11 +14,66 @@
 // You should have received a copy of the GNU General Public License
 // along with Cumulus.  If not, see <http://www.gnu.org/licenses/>.
 
-//! Cumulus test parachain collator
+//! Polkadot parachain-omni-node.
 
 #![warn(missing_docs)]
 #![warn(unused_extern_crates)]
 
+pub(crate) const EXAMPLES: &str = color_print::cstr!(
+	r#"<bold><underline>Examples:</></>
+
+   <bold>polkadot-parachain-omni-node --chain para.json --sync warp -- --chain relay.json --sync warp</>
+        Launch a warp-syncing full node of a given para's chain-spec, and a given relay's chain-spec.
+
+	<green><italic>The above approach is the most flexible, and the most forward-compatible used to spawn an omni-node.</></>
+
+	You can find the chain-spec of some networks in:
+
+	https://paritytech.github.io/chainspecs
+
+   <bold>polkadot-parachain-omni-node --chain asset-hub-polkadot --sync warp -- --chain polkadot --sync warp</>
+        Launch a warp-syncing full node of the <italic>Asset Hub</> parachain on the <italic>Polkadot</> Relay Chain.
+
+   <bold>polkadot-parachain-omni-node --chain asset-hub-kusama --sync warp --relay-chain-rpc-url ws://rpc.example.com -- --chain polkadot</>
+        Launch a warp-syncing full node of the <italic>Asset Hub</> parachain on the <italic>Kusama</> Relay Chain.
+        Uses <italic>ws://rpc.example.com</> as remote relay chain node.
+ "#
+);
+
+pub(crate) const BANNER: &str = color_print::cstr!(
+	r#"
+ _____                     _           _
+|  __ \                   | |         (_)
+| |__) |_ _ _ __ __ _  ___| |__   __ _ _ _ __
+|  ___/ _` | '__/ _` |/ __| '_ \ / _` | | '_ \
+| |  | (_| | | | (_| | (__| | | | (_| | | | | |
+|_|___\__,_|_|  \__,_|\___|_| |_|\__,_|_|_| |_|
+/ __ \                (_) | \ | |         | |
+| |  | |_ __ ___  _ __  _  |  \| | ___   __| | ___
+| |  | | '_ ` _ \| '_ \| | | . ` |/ _ \ / _` |/ _ \
+| |__| | | | | | | | | | | | |\  | (_) | (_| |  __/
+ \____/|_| |_| |_|_| |_|_| |_| \_|\___/ \__,_|\___|
+
+<i>
+Formerly know as polkadot-parachain, now called polkadot-parachain-omni-node, equipped with running more parachains in the Polkadot networks.
+</>
+
+Run with --help to see example usages.
+
+Please refer to the following resources to learn more about the omni-node:
+- https://forum.polkadot.network/t/polkadot-parachain-omni-node-gathering-ideas-and-feedback/7823/4
+- https://github.com/paritytech/polkadot-sdk/issues/5
+- sdk-docs
```

Comment:
I would like to work on a small set of ref-docs and push them to this PR about how to use the omni-node.

Reply:
I am trying to gain more attention to this. How about we let it spam all CLIs for a few versions, then move it to `--help`? 

I generally agree it is spammy. We can also move it to `--help` as of now.

Reply:
I removed it for the moment since it doesn't fit if we keep the old name. I can add it back if you decide that it's needed.

> How about we let it spam all CLIs for a few versions, then move it to --help ?

If we show it, we need to show it only just with `--help`. Otherwise for example the chain-spec generation commands will show it, resulting in malformed chain specs.